lib/Domain/Prototype/UseStatements.php

```diff
@@ -11,6 +11,9 @@ protected function singularName(): string
 
     public static function fromUseStatements(array $useStatements)
     {
+        usort($useStatements, function (UseStatement $left, UseStatement $right): int {
```

Comment:
Why do we need this?

Reply:
If not necessary, might be nicer to implement this as a method on the collection (e.g. `$useStatements = $useStatements->sorted()`)

Reply:
We need this when inserting multiple statements, because the new statements find where to go by looking at the initial list of use statements, not the up to date one.